assets/js/app.coffee

```diff
@@ -40,14 +40,15 @@ class Choosealicense
     # Dynamically add annotations as title attribute to rule list items
     for ruletype, rules of window.annotations
       for rule in rules
-        licenseLiElement = $(".license-#{ruletype} .#{rule["tag"]}")
+        # Only select license elements in table, not legend
+        licenseLiElement = $("td.license-#{ruletype} .#{rule["tag"]}")
```

Comment:
Belatedly noticing, I think this may have removed the tooltip from https://choosealicense.com/licenses/ and individual license pages.

Reply:
Good catch! Fixed in #1259